src/main/resources/reference-data/sscs-venues.csv

```diff
@@ -14,7 +14,7 @@
 37,SC037,SSCS Birmingham,Kettering Magistrates Court,The Court House,London Road,Kettering,,NN15 7QP,01536-410333,201,"https://www.google.com/maps/place/The+Court+House+London+Road+Kettering+NN15+7QP/@52.399079,-0.726940"
 38,SC038,SSCS Birmingham,Kings Court,,5A New Walk,,Leicester,LE1 6TE,01162-531730,201,"https://www.google.com/maps/place/52%C2%B037'48.5%22N+1%C2%B007'50.0%22W/@52.6301497,-1.1327347,17z/data=!3m1!4b1!4m5!3m4!1s0x0:0x0!8m2!3d52.6301497!4d-1.130546"
 40,SC040,SSCS Birmingham,Lincoln County Court Building,360 High Street,,Lincoln,Lincolnshire,LN5 7PS,01522-567151,201,"https://www.google.com/maps/place/Lincoln+County+Court/@53.2244617,-0.5450222,17z/data=!4m15!1m9!2m8!1sHotels!3m6!1sHotels!2s53.2244617,+-0.5428335!3s0x48785b2e4915e91f:0x7c0a2b187113f637!4m2!1d-0.5428335!2d53.2244617!3m4!1s0x48785b2e4915e8df:0xac30dad1e8201859!8m2!3d53.2243726!4d-0.5414549"
-43,SC043,SSCS Birmingham,The Church of Christ,The Cornerstone,300 Saxon Gate West,Central Milton Keynes,,MK9 2ES,,201,"https://www.google.com/maps/place/The+Cornerstone+300+Saxon+Gate+West+Central+Milton+Keynes+MK9+2ES/@52.041140,-0.761126"
+43,SC043,SSCS Birmingham,Milton Keynes Magistrates Court,301 Silbury Boulevard Witan Gate East,Central Milton Keynes,,MK9 2AJ,,201,"https://www.google.com/maps/place/Milton+Keynes+Magistrates'+Court/@52.0410469,-0.7684806"
```

Comment:
is it missing a comma between `Boulevard` and `Witan`

Reply:
should the google maps url work when you paste it in a browser?

Reply:
Thanks now its fixed